### PR TITLE
more responsive UI

### PR DESCRIPTION
**Pull request by HoLuDyEr**
This PR adds a more responsive UI for different screen sizes

This also changes to size of the graph by using the viewbox attribute

some examples:

image

image

image

image

hard to see like this but they have a large difference in width

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -6,6 +6,63 @@
 #fp-nav ul li .fp-tooltip {
     color: black;
 }
+
+/* left and right panels ---------------------------------------------------------- */
+.container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    height: 99%;
+    width: 99%;
```
Comment (HugoHof) [inline css/style.css:15]: By curiosity, why 99% ?

File: css/style.css [hunk 2]
```diff
@@ -124,14 +181,14 @@ h2 {
...
     height: 100%;
     padding: 0%;
-    margin: 0%;
+    margin: 0%;*/
     background-color: #000000;
-    align-items: center;
-    align-content: center;
+    /*align-items: center;
+    align-content: center;*/
 }
```
Comment (HugoHof) [inline css/style.css:192]: Value commented I think, I don't know if it is normal.

File: index.html [hunk 3]
```diff
@@ -201,7 +203,7 @@ <h1 class="gameCardTitle" id="games_genre_rank_3_title">Title #3</h1>
                                             </div>
                                         </div>
     
-                                        <div style="height: 55px; background-color: #CD7F32;" class="podiumStep">3</div>
+                                        <div style="background-color: #CD7F32;" class="podiumStep">3</div>
```
Comment (HugoHof) [inline index.html:206]: The problem that there is no difference between the 1st, 2nd and third place, You have to let the size in the style here.

File: index.html [hunk 4]
```diff
@@ -182,7 +184,7 @@ <h1 class="gameCardTitle" id="games_genre_rank_1_title">Title #1</h1>
                                             </div>
                                         </div>
     
-                                        <div style="height: 95px; background-color: #ffd700;" class="podiumStep">1</div>
+                                        <div style="background-color: #ffd700;" class="podiumStep">1</div>
```
Comment (HugoHof) [inline index.html:187]: The problem that there is no difference between the 1st, 2nd and third place, You have to let the size in the style here.

File: index.html [hunk 5]
```diff
@@ -163,7 +165,7 @@ <h1 class="gameCardTitle" id="games_genre_rank_2_title">Title #2</h1>
                                             </div>
                                         </div>
     
-                                        <div style="height: 75px; background-color: #C0C0C0;" class="podiumStep">2</div>
+                                        <div style="background-color: #C0C0C0;" class="podiumStep">2</div>
```
Comment (HugoHof) [inline index.html:168]: The problem that there is no difference between the 1st, 2nd and third place, You have to let the size in the style here.

**Review by HugoHof — CHANGES_REQUESTED** [review]
All Good. There is just a problem with the podium in the Genre diagram . We can also see it on the images.

**Comment by HugoHof** [comment]
> All Good. There is just a problem with the podium in the Genre diagram . We can also see it on the images.

Other problem, the size of the game icon in the list of games in the developer's visualization haven't the same size like before.

File: css/style.css (hunk 1 above)
Comment (HoLuDyEr) [reply css/style.css:15]: because fullpage.js has a 1% left margin, this compensate

**Comment by HoLuDyEr** [comment]
now the game cards are fixed, they have the maximum height possible (all the same height) and can overflow vertically if needed 

image

here the top 3 placement is fixed

image

File: index.html (hunk 4 above)
Comment (HoLuDyEr) [reply index.html:187]: fixed

File: css/style.css (hunk 2 above)
Comment (HoLuDyEr) [reply css/style.css:192]: it is

**Comment by HugoHof** [comment]
> now the game cards are fixed, they have the maximum height possible (all the same height) and can overflow vertically if needed
> 
> image
> 
> here the top 3 placement is fixed
> 
> image

It was the step of the podium which has a different size, not the relative position of the podium. The step podium must be aligned.

**Comment by HoLuDyEr** [comment]
like this?

image

**Comment by HoLuDyEr** [comment]
2 fixes, now the top 3 have different heights and the dropdown menu is not on the graph:

image

image

**Comment by HugoHof** [comment]
> like this?
> 
> image
Capture d’écran 2024-05-25 à 22 23 46
The position of the podium was modify, in comaprison with the previous one (see image)

**Review by HugoHof — APPROVED** [review]
All Good

**Pull request merged**